plutus-benchmark/bls12-381-costs/src/PlutusBenchmark/BLS12_381/RunTests.hs

```diff
@@ -84,9 +79,41 @@ runTests h = do
 
   hPrintf h "Groth16 verification example\n\n"
   printHeader h
-  printCosts_Groth16Verify h
+  printSizeStatistics h NoSize mkGroth16VerifyScript
   hPrintf h "\n"
 
   if checkGroth16Verify_Haskell
   then hPrintf h "Groth16Verify succeeded\n"
   else hPrintf h "Groth16Verify failed\n"
+
+  if checkVerifyBlsSimpleScript
```

Comment:
I think you could get it to print out the costs of all of the new tests just by mimicking what it does for Groth16Verify above.  That might be interesting to see...

Reply:
Done. It exposed that the test that performs multi-key aggregation goes over CPU budget when 10 keys are used.